### DISCUSSION TITLE
tailwind.config.js path, for Django

### DISCUSSION THREAD

**Discussion by j2l** [tailwindlabs/tailwindcss#13930] [Help]
Hello,
Doc (https://tailwindcss.com/docs/configuration) states that `tailwind.config.js` is searched at the root of a project.
I have it available at `http://127.0.0.1:8000/tailwind.config.js` and `http://127.0.0.1:8000/static/tailwind.config.js` with a custom theme (mytheme) and it is not seen by base.html:
```html
<html data-theme="mytheme"></html>
<head>
...
<link href="/static/full4_12_8.min.css" rel="stylesheet" type="text/css" />
<script src="https://cdn.tailwindcss.com"></script> 
```
Only default DaisyUI with Tailwind is visible on the page.

tailwind.config.js is:
```js
module.exports = {
daisyui: {
    themes: [
      {
        mytheme: {
"primary": "#...
...
"error": "#...",
          },
        },
      ],
    },
}
```
If I change to cupcake, it does switch, but I'd like my custom theme :)

Would you please let me know the list of paths it looks for?
And does the cdn script looks on my project or elsewhere?

Thank you!

**Comment by wongjn** [comment]
> Doc (https://tailwindcss.com/docs/configuration) states that `tailwind.config.js` is searched at the root of a project.

This does not apply to your circumstance since you are using the CDN. When using the CDN, the configuration is at the `tailwind` global object.

Furthermore, I do not believe custom Daisy UI configuration will work with CDN versions. You'd need to compile the CSS yourself using Tailwind CLI or integrated in a PostCSS build system.

**Comment by j2l** [comment]
Thank you @wongjn !
It makes more sense now.

> integrated in a PostCSS build system.

No idea how to do this in python dev ecosystem. Any idea?

**Reply by wongjn** [reply]
If you have some build system in your project already for compiling assets, you could see if one of the [framework guides](https://tailwindcss.com/docs/installation/framework-guides) is applicable to you. Otherwise, you could consider using [the Tailwind CLI](https://tailwindcss.com/docs/installation).
